DEVELOPMENT_USAGE.md

```diff
@@ -0,0 +1,143 @@
+# Development Usage Notes
+
+> How to use operations in other projects
+
+Operations is both a directory hierarchy containing metadata related to beacons and also a set of libraries to read,
+validate, normalize, write and manipulate that data.
+
+Common use cases are reading operations as a JSON payload and display that data in a UI in some way.
+
+Below are examples of how to achieve that and a general demonstration of the relationships between data.
+
+### The following commands assume an empty directory and the creation of a project
+
+```shell
+# Initialise the new project
+yarn init
+# You'll be prompted to enter various info about your project
+
+# Install the published operations package
+yarn add @api3/operations
+
+# You may also want Typescript
+yarn add typescript ts-node @types/node -D
+```
+
+Create a demo application file to execute:
+
+```shell
+mkdir scripts
+touch scripts/export.js
+```
+
+Edit scripts/export.js
+
+Import the dependency into your application:
+
+```javascript
+const { readOperationsRepository } = require('@api3/operations/dist/utils/read-operations');
```

Comment:
Nit: we may want to export all functions under `/utils` in an indext.ts on the root to make the import path shorter

Reply:
Good point. I've done this but importing using the index instead of importing `read-operations` directly introduces a need for additional dependencies. I'm not entirely sure how to handle this; most usage of operations will be consumption using `read-operations`, `write-operations` will only be used during development.